nautobot_floor_plan/forms.py

```diff
@@ -27,6 +27,17 @@ class FloorPlanForm(NautobotModelForm):
 
     location = DynamicModelChoiceField(queryset=Location.objects.all())
 
+    x_origin_seed = forms.CharField(
+        label = "Beginning X value",
+        help_text = "The first value to begin X Axis at.",
+        required=False,
+    ) 
+    y_origin_seed = forms.CharField(
+        label = "Beginning Y value",
```

Comment:
```suggestion
        label = "Y Axis Seed",
```

Making this consistent with the detail view.